pkg/routes/linux_routes.go

```diff
@@ -30,3 +32,109 @@ func DeleteByDestination(destinationSubnet *net.IPNet) error {
 	}
 	return nil
 }
+
+func InjectRoute(subnet *net.IPNet, gw net.IP, ipa utils.NodeIPAware, tn pkg.Tunneler, rs pkg.RouteSyncer,
```

Comment:
@twz123 - Now that this logic is disentangled with the BGP logic that used to be buried in here I like it a lot better. It feels a lot cleaner and more straight forward to me.